flavors/liquid-mainnet/extras.css

```diff
@@ -1,5 +1,7 @@
 
-
+.sub-nav .active{
+  border-bottom: solid 2px rgba(40, 148, 143, 1);
```

Comment:
Why rgba with opacity of 1? (here and elsewhere)

My preference would be to stick with #rrggbb.

Reply:
Changes have been made